src/Breadcrumb/index.d.ts

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export interface BreadcrumbProps {
+  links: Array<{
+    label: string,
+  }>,
+  activeLabel?: string,
+  spacer?: React.ReactElement,
+  clickHandler?: Function,
```

Comment:
Let's clarify that this function does not return anything (for example: `clickHandler?: () => void`)